scripts/fixup_check

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-output=$(git log --oneline | grep -iE 'fixup|wip')
+output=$(git log --oneline --since="15/10/2019" | grep -iE 'fixup|wip')
```

Comment:
We could also do `git log --oneline origin/fleming.. | grep ...` to only apply the check on the commits added in the current PR. Would have to remember to change it to `origin/master` when we merge fleming though.

Reply:
Tried it:

```
git log --oneline origin/fleming.. | grep -iE 'fixup|wip'
9fec248310 fixup
019891a8a8 fixup! feat/chain: increase members' age on churn events
1fcd2bf56b fixup! feat/chain: properly initialise the age counter
d0dcef0fee fixup! feat/chain: increase members' age on churn events
f7b5c839d4 fixup! feat/chain: properly initialise the age counter
adaa2e3a9e fixup! chore/node: rename section_members to section_elders
```

Reply:
I presume `origin` here refers to the MaidSafe repo and not your own fork?

Reply:
Yes, I think when that is run by the CI server then `origin` would refer to the maidsafe repo. To try it locally, use just `fleming` (or `upstream/fleming` or whatever your local name for the maidsafe repo is)

Reply:
Well `git log --oneline origin/fleming.. | grep -iE 'fixup|wip'` seems to work as intended when I tried it locally

Reply:
Was thinking that maybe the reason it didn't work for @jeanphilippeD was that his own `origin` pointed to his own fork?

Reply:
That works better with git log --oneline upstream/fleming.. | grep -iE 'fixup|wip'

Reply:
What happens if something is merged to fleming though will we still see the fixup?

Reply:
> What happens if something is merged to fleming though will we still see the fixup?
Not sure I follow, `origin/fleming..` should list all commits not merged